packages/storage/Readme.md

```diff
@@ -40,7 +40,7 @@ And initialize it using the following lines:
 pub static OWNER: Item<HumanAddr> = Item::new(b"owner");
 ```
 
-This uses Bincode2 to serde HumanAddr by default. To specify the Serde algorithm as Json, first import it from `secret-toolkit-serialization`
+This uses Bincode2 to serde HumanAddr by default. To specify the Serde algorithm as Json, first import it from `secret-toolkit::serialization`
```

Comment:
We should update the README to `Addr` and `String` instead of `HumanAddr`.
Maybe Ctrl+F and replace every HumanAddr occurrence with `Addr`?